Frontend/CarRentalAdmin/src/app/_services/auth.service.ts

```diff
@@ -0,0 +1,72 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable } from 'rxjs';
+import { tap } from 'rxjs/operators';
+
+import { LoginRequest } from '../_models/login-request';
+import { AuthResponse } from '../_models/auth-responce';
+import { RefreshTokenRequest } from '../_models/refresh-token-request';
+
+import { TokenService } from './token.service';
+
+@Injectable()
+export class AuthService {
+    private loggedIn: boolean = false;
```

Comment:
do you remember we discussed observable approach? what do you think of using it here?